_includes/parse-server/graphql.md

```diff
@@ -0,0 +1,325 @@
+# GraphQL
+
+[GraphQL](https://graphql.org/), developed by Facebook, is an open-source data query and manipulation language for APIs. In addition to the traditional REST API, Parse Server automatically generates a GraphQL API based on your current application schema. Parse Server also allows you to define your custom GraphQL queries and mutations, whose resolvers can be bound to your cloud code functions.
+
```

Comment:
This is a rather long section on graphql, especially considering the short section on the rest api.

It might be worth looking at what the crossover with the GraphQL guide is.

Any thoughts on this appreciated...

Reply:
I do agree it could be condensed dramatically

Reply:
I think most of this file should be removed to leave something similar to what is present for REST - including the parts on getting your Parse GraphQL Server up and running and, then maybe leave the part about the API health check and link to the GraphQL guide for the rest.

I couldn't see anything on `Using Docker` in the GraphQL guide so perhaps there are a couple bits to add in there.